exercises/triangle/src/main/java/Triangle.java

```diff
@@ -4,8 +4,4 @@
         throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
     }
 
-    TriangleKind getKind() {
-        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
-    }
```

Comment:
Please add stubs for the new methods. The policy for starter implementation is in the [POLICIES doc](https://github.com/exercism/java/blob/master/POLICIES.md#starter-implementations). This is the 17th exercise in the track by difficulty (from the order in `config.json`) so it should have stubs for all required methods :)

Reply:
I did see your comment asking about stubs and it's perfectly fine that you weren't sure of course :) The documentation for things can be a bit hard to find sometimes :P